pdns/slavecommunicator.cc

```diff
@@ -1182,7 +1182,11 @@ void CommunicatorClass::slaveRefresh(PacketHandler *P)
 
       DomainNotificationInfo dni;
       dni.di=di;
-      dni.dnssecOk = dk.doesDNSSEC();
+      if (::arg().mustDo("compare-signatures-on-zone-freshness-check")) {
```

Comment:
Checking this over and over again inside the loop is a waste of resources since the value will never change.

Reply:
indeed. addressed, thanks.